pkg/backup/backup.go

```diff
@@ -240,6 +243,11 @@ func (kb *kubernetesBackupper) Backup(log logrus.FieldLogger, backupRequest *Req
 	log.Infof("Including resources: %s", backupRequest.ResourceIncludesExcludes.IncludesString())
 	log.Infof("Excluding resources: %s", backupRequest.ResourceIncludesExcludes.ExcludesString())
 
+	if backupRequest.Backup.Spec.DefaultRestic == nil {
+		backupRequest.Backup.Spec.DefaultRestic = &kb.defaultRestic
+	}
+	log.Infof("Backing up all pod volumes using restic: %t", *backupRequest.Backup.Spec.DefaultRestic)
```

Comment:
I'd probably move this logic into the backup controller where we have some other defaulting -- somewhere in the [prepareBackupRequest](https://github.com/vmware-tanzu/velero/blob/d9d9dc60da8e82a205305d4fe54daf36e2f40f4a/pkg/controller/backup_controller.go#L318) method. It should all move out of the controller eventually and into a library, but this'll keep similar logic together.

Reply:
done!